Backoffice login

hafidnrzs opened the pull request:
## What did you do?

- Implementasi login untuk Backoffice
- Atur ulang styling helper text Input Field

Lain-lain:
- Ubah favicon untuk Backoffice

hafidnrzs committed:
feat(backoffice): send login request

- buat .env untuk menyimpan API dan memberikan .env.example juga untuk contohnya. Tambah .gitignore juga
- tambah fungsi login pada Backoffice

hafidnrzs committed:
feat(backoffice): implement login and logout

- Tambah fitur login dan logout pada Backoffice
- Ubah port development server dan atasi CORS
- Update type login response
- Update component Input agar toggle password berfungsi dengan benar

hafidnrzs committed:
Merge branch 'develop' into feat/backoffice-login

hafidnrzs committed:
Merge branch 'develop' into feat/backoffice-login

hafidnrzs committed:
feat: login backoffice

maulanasdqn commented on libs/service/src/api/auth/index.ts:
```diff
@@ -12,7 +12,7 @@ export const postLogin = async (
   const { data } = await api({
     method: 'POST',
-    url: '/auth/login',
+    url: '/v1/auth/login',
```
v1 gak perlu ada disini

cukup di base url nya aja

jadi di env VITE_API_URL=https://api.imphnen.dev/v1

maulanasdqn commented on libs/ui/src/organisms/backoffice-sidebar/backoffice-sidebar.tsx:
```diff
@@ -14,6 +14,13 @@ export const BackofficeSidebar: FC = (): ReactElement => {
 
+  const handleLogout = () => {
+    sessionStorage.removeItem('access_token');
+    localStorage.removeItem('refresh_token');
```
helper nya udh ada, tinggal panggil useSession

hafidnrzs committed:
Merge branch 'develop' into feat/backoffice-login

hafidnrzs committed:
refactor auth and logout

- Pindah hard-code API version ke environment variable
- Gunakan helper untuk logout
